cmis/src/main/java/nl/nn/adapterframework/extensions/cmis/CmisSender.java

```diff
@@ -941,6 +940,22 @@ private Message sendMessageForDynamicActions(Session cmisSession, Message messag
 		return new Message(resultXml.toXML());
 	}
 
+	protected CmisEvent parseEvent(String cmisEvent) {
+		if(StringUtils.isNotEmpty(cmisEvent)) {
+			try {
+				return EnumUtils.parseDocumented(CmisEvent.class, "CmisEvent", cmisEvent);
+			} catch (IllegalArgumentException e1) {
+				try {
+					return EnumUtils.parseNormal(CmisEvent.class, "CmisEvent", cmisEvent);
+				} catch (IllegalArgumentException e2) {
+					e1.addSuppressed(e2);
+					throw e1;
+				}
```

Comment:
Hier doet hij denk ik eerst via 'enum label' proberen te parsen, en als dat niet lukt, via 'name'. 
Als dat zo is, dan zou dat gedrag ook in EnumUtils zelf kunnen zitten, via een aparte method, of via een optie in parseDocumented()